tests/gis_tests/inspectapp/tests.py

```diff
@@ -28,7 +28,7 @@ def test_geom_columns(self):
         """
         out = StringIO()
         call_command('inspectdb',
```

Comment:
You might care to reformat these function calls to use the normal 4 space hanging indent style.